pyrail/models.py

```
@@ -19,20 +20,114 @@
+
+
+@dataclass
+class Segment(DataClassORJSONMixin):
+    id: str  # ID of the segment
+    origin: Station  # Origin station information
+    destination: Station  # Destination station information
+    composition: SegmentComposition  # Composition details of the segment
+
+
+@dataclass
+class CompositionSegments(DataClassORJSONMixin):
```

Ruff (0.8.2)

Missing docstring in public class

(D101)